catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/content/brick/stage/SayForBubbleBrickStageTest.java

```diff
@@ -80,7 +84,7 @@ public void sayForBubbleBrickStageTest() {
 		assertNull(StageActivity.stageListener.getBubbleActorForSprite(sprite));
 		onView(isFocusable())
 				.perform(click());
-		assertNotNull(StageActivity.stageListener.getBubbleActorForSprite(sprite));
+		assertTrue(waitForBubbleActorForSprite(2000));
```

Comment:
you could wait on the root view instead of using a stopwatch.
```suggestion
              onView(ViewMatchers.isRoot()).perform(CustomActions.wait(1000)) // 1 sec would be enough
              assertNotNull(StageActivity.stageListener.getBubbleActorForSprite(sprite));
```

Reply:
Done. Thanks for your Feedback!